### PR TITLE
adjust run_if_changed for serverless, keda-manager and warden jobs

### PR DESCRIPTION
**Pull request by anoipm**
**Description**

Changes proposed in this pull request:

- adjust (simplify) run_if_changed for serverless, keda-manager and warden jobs

### CODE REVIEW

**Review by Ressetkk — DISMISSED** [review]

File: templates/data/keda-manager.yaml [hunk 1]
```diff
@@ -33,6 +33,8 @@ templates:
           limits:
             limits_memory: 3Gi
             limits_cpu: 2
+          run_if_changed_without_documentation:
+            skip_if_only_changed: '^docs/|^examples/|\.md$'
```
Comment (pPrecel) [inline templates/data/keda-manager.yaml:37]: Good idea 👍🏿

File: templates/data/keda-manager.yaml [hunk 2]
```diff
@@ -33,6 +33,8 @@ templates:
           limits:
             limits_memory: 3Gi
             limits_cpu: 2
+          run_if_changed_without_documentation:
```
Comment (pPrecel) [inline templates/data/keda-manager.yaml:36]: I would suggest renaming it to something related to the `skip_if_only_changed` field. For example:
* `skip_if_only_changed_documentation`

Comment (anoipm) [reply templates/data/keda-manager.yaml]: done

**Review by pPrecel — APPROVED** [review]

**Review by Ressetkk — APPROVED** [review]

**Pull request merged**